statsmodels/stats/proportion.py

```diff
@@ -569,6 +569,9 @@ def proportions_ztest(count, nobs, value=None, alternative='two-sided',
     TODO: add continuity correction or other improvements for small samples.
 
     '''
+    count=np.asarray(count)
+    nobs=np.asarray(nobs)
```

Comment:
@josef-pkt Sorry i forgot to put space around = , i will keep that in mind .
What if the nobs.size is not equal to 1 , and the also the shape of counts and nobs are different . So should it raise a ValueError ??
Also i was getting error while trying to vectorize it . 
So should i make a PR by adding if condition when nobs.size=1 and raising error when shape is different of count and nobs ??